src/Client/Player/Player.py

```diff
@@ -19,15 +19,26 @@ class PicFile(Enum):
 
 class Player(pg.sprite.Sprite):
 
-    def __init__(self, player_id, pic_file):
+    def __init__(self, player_id, name, pic_file):
         super(Player, self).__init__()
         self.logger = logging.getLogger('Domi.Player')
         try:
-            self.surf = pg.image.load(pic_file.value).convert()  # may have to change the path
+            img_surf = pg.image.load(pic_file.value).convert()  # may have to change the path
+            self.surf = pg.Surface((60, 60))
+            self.surf.fill((255, 253, 201))
```

Comment:
Khm... random numbers, which are coincidentally are the same as some lines below

Reply:
Yep, but it just works. ;)

Reply:
Yes, until we change one of them...